README.md

```diff
@@ -170,13 +170,16 @@ rake build:all
 
 ## Hosting
 
-Before you host your app in a production environment, make sure to create the production app in your Partners Dashboard.
-You'll need to set up the API key and API secret for your production environment, as per the instructions below.
+Once you're ready to set up your app in production, you can follow [our deployment documentation](https://shopify.dev/apps/deployment) to host your app on a cloud provider like [Heroku](https://www.heroku.com/) or [Fly.io](https://fly.io/).
 
-The following pages document the basic steps to host and deploy your application to a few popular cloud providers:
+**Note**: Once you reach the step for [setting up environment variables](https://shopify.dev/apps/deployment/web#step-3-set-environment-variables) in the docs, you will also need to set these variables:
```

Comment:
I'd pull these out of a note because it's important. also use this anchor so this link doesn't break: https://shopify.dev/apps/deployment/web#set-env-vars

```suggestion
When you reach the step for [setting up environment variables](https://shopify.dev/apps/deployment/web#set-env-vars), you also need to set the following variables:
```